docs/api-status.md

```diff
@@ -13,6 +13,7 @@
 Name | Added in Version | Expected Stable Version | 
 ---- | ---------------- | ----------------------- | 
 WriteOp.CmpExt | v0.12.0 | v0.14.0 | 
+ReadOp.Read | v0.12.0 | v0.14.0 | 
```

Comment:
Same here. This file is generated based on the json - you can either update this by hand or by running the apiage tool. For a trivial edit the by-hand appraoch is probably faster :-)